lib/pbench/server/database/models/tracker.py

```diff
@@ -396,6 +399,26 @@ def validate_owner(self, key: str, value: Any) -> User:
                 return user
         raise DatasetBadParameterType(value, "username")
 
+    @validates("access")
+    def validate_access(self, key: str, value: str) -> str:
+        """
+        Validate the access key for the dataset.
+
+        Args:
+            key: access
+            value: string "private" or "public"
+
+        Raises:
+            DatasetBadParameterType: the access value given isn't allowed.
+
+        Returns:
+            access keyword string
+        """
+        access = value.lower()
+        if access == "public" or access == "private":
```

Comment:
I don't know whether it's better, but I think it's "more Pythonic" to express this as
```
        if access in ["public", "private"]:
```

Reply:
In fact, I really should (have) put `ACCESS_KEYS = [ "public", "private" ]` at the class level and test against that...